test/blocks/tiktok/tiktok.test.html

```diff
@@ -0,0 +1,42 @@
+<html>
+  <head>
+    <!-- the import map to use in our test -->
+    <script type="importmap">
+      {
+        "imports": {
+          "../../../libs/utils/utils.js": "./mocks/utils.js"
+        }
+      }
+    </script>
+  </head>
+
+  <body>
+    <a href="https://www.tiktok.com/@adobe/video/7171880842296347946">https://www.tiktok.com/@adobe/video/7171880842296347946</a>
+  </body>
+
+  <script type="module">
+    import { runTests } from '@web/test-runner-mocha';
+
+    import { readFile } from '@web/test-runner-commands';
+    import { expect } from '@esm-bundle/chai';
+    import { loadScript } from '../../../libs/utils/utils.js';
+    const { default: init } = await import('../../../libs/blocks/tiktok/tiktok.js');
+
+    runTests(() => {
+      describe('tiktok', () => {
+        it('renders embed video for tiktok link', async () => {
+          const tiktok = document.querySelector('a');
+
+          init(tiktok);
+
+          const blockquote = document.querySelector('.tiktok-embed');
+          const script = document.querySelector('script[src="https://www.tiktok.com/embed.js"]');
+
```

Comment:
every other test does not have a newline between theses `const` declarations and the expects below. I suppose it doesn't matter too much, but might be nice to be consistent across files.

Reply:
Made sure all the tests have consisntent spacing, at least all the embed ones I've touched